src/main/kotlin/com/terraformation/backend/accelerator/db/ApplicationStore.kt

```diff
@@ -108,6 +119,130 @@ class ApplicationStore(
     }
   }
 
+  fun fetchModulesByApplicationId(applicationId: ApplicationId): List<ApplicationModuleModel> {
+    requirePermissions { readApplication(applicationId) }
+
+    return with(MODULES) {
+      dslContext
+          .select(
+              asterisk(),
+              APPLICATION_MODULES.APPLICATION_ID,
+              APPLICATION_MODULES.APPLICATION_MODULE_STATUS_ID)
+          .from(this)
+          .join(APPLICATION_MODULES)
+          .on(APPLICATION_MODULES.MODULE_ID.eq(ID))
+          .where(DSL.or(PHASE_ID.eq(CohortPhase.PreScreen), PHASE_ID.eq(CohortPhase.Application)))
```

Comment:
This is totally fine. FYI, there are a couple other (equally fine, slightly shorter) ways to do this, in case you end up preferring one of them.

Chained function call for OR (nested inside the `where()` so it has the same effect as the `DSL.or()` call):
```
.where(PHASE_ID.eq(CohortPhase.PreScreen).or(PHASE_ID.eq(CohortPhase.Application)))
```
IN (needs backticks because `in` is a Kotlin keyword):
```
.where(PHASE_ID.`in`(CohortPhase.PreScreen, CohortPhase.Application))
```